modeler.js

```diff
@@ -0,0 +1,131 @@
+import axios from "axios";
+import { getAccessToken } from "./auth.js";
+
+const authorizationConfiguration = {
+  clientId: process.env.MODELER_CLIENT_ID,
+  clientSecret: process.env.MODELER_CLIENT_SECRET,
+  audience: process.env.MODELER_AUDIENCE
+};
+
+// An action that creates a new project.
+async function createProject([projectName]) {
```

Comment:
non-blocking conversation: 

One thing I learned while testing this, and it is documented in the Modeler API's swagger UI -- when creating a new project through the API, it has no collaborators, and therefore can't be viewed through the Modeler UI until you add collaborators via a different API endpoint. I discovered this, because I couldn't find my project in the UI after creating it 😅

I'm not sure what that means for this function. Do you think it's fine that the user won't be able to find their project in the UI after creating it? _I_ tried to find mine; I imagine others might do the same. Should we add an argument to this function for the collaborator and make a call to the second endpoint for them after creating the project? Are projects maybe not the panacea that I thought they were, and we should look for _another_ resource that can be created with no extra effort?